### PR TITLE
UIROLES-64: Duplicate authorization roles

### PR DESCRIPTION
**Pull request by alisher-epam**
Refs [UIROLES-64](https://folio-org.atlassian.net/browse/UIROLES-64) - Add `Duplicate` authorization role capability for the Authorization role settings.

https://github.com/user-attachments/assets/f708ff10-b196-496d-8904-e0d904c4c241

### CODE REVIEW

File: src/settings/SettingsPage/SettingsPage.js [hunk 1]
```diff
@@ -20,16 +20,52 @@ import {
...
+  const {
+    duplicateAuthorizationRole,
+    isLoading: isDuplicating,
+  } = useAuthorizationRolesMutation();
+
+  const { roleDetails } = useRoleById(roleId);
+
+  const onDuplicate = () => {
+    const roleName = roleDetails?.name;
+    const messageIdPrefix = 'ui-consortia-settings.consortiumManager.members.authorizationsRoles.duplicate';
```
Comment (zburke) [inline src/settings/SettingsPage/SettingsPage.js:50]: Is role-duplication only available within ECS? If so, the menu item needs to be fenced off by an `<IfInterface>`. If not, then this translation needs to be moved from the `ui-consortia-settings` repo into this one.

**Review by zburke — CHANGES_REQUESTED** [review]
Code changes look OK, except (most likely) the translation needs to move out of `ui-consortia-settings` into this repo since non-ECS tenants will not have that module included in their builds.

File: src/settings/SettingsPage/SettingsPage.js (hunk 1 above)
Comment (alisher-epam) [reply src/settings/SettingsPage/SettingsPage.js]: added translation message here

Comment (alisher-epam) [reply src/settings/SettingsPage/SettingsPage.js]: It is available for both ECS and non-ECS

**Review by zburke — APPROVED** [review]

**Pull request merged**